app/assets/stylesheets/publisher.scss

```diff
@@ -98,6 +98,12 @@
         line-height: $line-height-base !important;
       }
 
+      .twitter-typeahead {
+        position: absolute !important;
```

Comment:
No choice here: only way to override the CSS rules dynamically added to the element.